src/tools/rbd/action/Trash.cc

```diff
@@ -151,6 +150,8 @@ std::string delete_status(time_t deferment_end_time) {
   std::stringstream ss;
   if (now < deferment_end_time) {
     ss << "protected until " << time_str;
+  } else {
+    ss << "expired at      " << time_str;
```

Comment:
Nit: lots of extra spaces

Reply:
it results in clean readable output, imagine many "expired at" and "protected until" one after another
```
[11:06][root@xxx (ceph_dev:ceph/build*10) build]# bin/rbd trash list --pool mypool --long
ID           NAME SOURCE DELETED_AT               STATUS                                   
10d174b0dc51 bar  USER   Thu Oct 12 10:51:23 2017 expired at      Fri Oct 13 10:51:23 2017 
10d72ae8944a faz  USER   Wed Oct 18 11:06:08 2017 protected until Fri Oct 20 11:06:08 2017 
10eb2ae8944a foo  USER   Mon Oct 16 11:25:10 2017 expired at      Mon Oct 16 11:25:10 2017 
11032ae8944a baz  USER   Wed Oct 18 11:01:02 2017 protected until Thu Oct 19 11:01:02 2017 
11262ae8944a laz  USER   Wed Oct 18 11:05:19 2017 expired at      Tue Oct 17 11:05:19 2017 
```

Reply:
... true, but if using JSON/XML formatted output, the spacing is unnecessary.

Reply:
I am sorry, thanks again!

Reply:
No worries -- thx